tests/test_mumps.py

```diff
@@ -0,0 +1,3516 @@
+# -*- coding: utf-8 -*-
+"""
+    Basic MumpsLexer Test
```

Comment:
This test file is a lot of code and uses exactly the same repetitive pattern everywhere. Because this is what most lexers do, we have infrastructure for it. Please put each test in a file under `tests/snippets/mumps` and use `pytest --update-goldens tests/snippets/mumps` to generate the token output.

Reply:
I should have looked harder! As soon as I found tests/test_*.py I assumed that was the place. I'll get these cleaned up next.

Reply:
Lexer tests have been moved entirely over to snippets.